flang/runtime/extensions.cpp

```diff
@@ -13,6 +13,44 @@
 #include "flang/Runtime/command.h"
 #include "flang/Runtime/descriptor.h"
 #include "flang/Runtime/io-api.h"
+#include <string.h>
```

Comment:
We should use `cstring` instead of `string.h` and make sure all C standard functions are prefixed with `std::` since this is a C++ file